client/css/style.css

```diff
@@ -149,8 +144,22 @@ kbd {
 	opacity: 1;
 }
 
+input[type="checkbox"],
+input[type="radio"] {
+	transition: box-shadow 0.2s;
+}
+
+#chat .search:focus,
+.input:focus,
```

Comment:
I removed the `#window` part of this, because "Join a channel" form is not part of it and therefore was not being picked up.